client/spring-zeebe/src/main/java/io/camunda/zeebe/spring/client/jobhandling/SpringConnectorJobHandler.java

```diff
@@ -3,30 +3,44 @@
 import io.camunda.connector.api.error.BpmnError;
 import io.camunda.connector.api.outbound.OutboundConnectorFunction;
 import io.camunda.connector.api.secret.SecretProvider;
+import io.camunda.connector.impl.outbound.OutboundConnectorConfiguration;
 import io.camunda.connector.runtime.util.outbound.ConnectorJobHandler;
 import io.camunda.connector.runtime.util.outbound.ConnectorResult;
 import io.camunda.zeebe.client.api.response.ActivatedJob;
 import io.camunda.zeebe.client.api.worker.JobClient;
+import io.camunda.zeebe.spring.client.connector.MetricsRecorder;
 
 public class SpringConnectorJobHandler extends ConnectorJobHandler {
 
   private final CommandExceptionHandlingStrategy commandExceptionHandlingStrategy;
+  private final OutboundConnectorConfiguration connectorConfiguration;
+  private final MetricsRecorder metrics;
 
-  public SpringConnectorJobHandler(OutboundConnectorFunction call, SecretProvider secretProvider,
-    CommandExceptionHandlingStrategy commandExceptionHandlingStrategy) {
-    super(call, secretProvider);
+  public SpringConnectorJobHandler(OutboundConnectorConfiguration connectorConfiguration, SecretProvider secretProvider,
+                                   CommandExceptionHandlingStrategy commandExceptionHandlingStrategy, MetricsRecorder metrics) {
+    super(connectorConfiguration.getFunction(), secretProvider);
+    this.connectorConfiguration = connectorConfiguration;
+    this.metrics = metrics;
     this.secretProvider = secretProvider;
     this.commandExceptionHandlingStrategy = commandExceptionHandlingStrategy;
   }
 
+  @Override
+  public void handle(JobClient client, ActivatedJob job) {
+    metrics.increaseActivated(MetricsRecorder.METRIC_NAME_OUTBOUND_CONNECTOR, connectorConfiguration.getType());
```

Comment:
```suggestion
    metrics.increaseActivated(MetricsRecorder.METRIC_NAME_JOB, connectorConfiguration.getType());
```

Reply:
The original code is correct /intended. The ConnectorJobHandler is only called for connectors - so it should count connectors - I just wondered if we should maybe use the connector name instead of the type (but then we would miss the version).